amical/tools.py

```diff
@@ -368,10 +369,11 @@ def jd2lst(lng, jd):
     return lst
 
 
-def compute_pa(hdr, n_ps, verbose=False, display=False):
+def compute_pa(hdr, n_ps, sci_hdr, verbose=False, display=False):
```

Comment:
new arguments shouldn't be added before the last allowed-positional existing one (here, `display`) to preserve backward compatibility. I suggest making this one keyword only and put it at the end of the signature. Since `verbose` and `display` have default values this one will need one too.

Reply:
I updated the signature and calls to make sci_hdr keyword-only